models/unit.go

```diff
@@ -137,3 +141,13 @@ var (
 		UnitTypeExternalWiki:    UnitExternalWiki,
 	}
 )
+
+// FindUnitTypes give the unit key name and return unit
+func FindUnitTypes(nameKeys ...string) (res []UnitType) {
+	for t, u := range Units {
+		if strings.EqualFold(u.NameKey, u.NameKey) {
```

Comment:
This is always `true`. You are comparing `u.NameKey` with `u.NameKey`...

Reply:
done.